gamemode/modules/language/sh_language.lua

```diff
@@ -24,6 +24,13 @@ function DarkRP.getPhrase(name, ...)
     return (langTable[name] or rp_languages.en[name]) and string.format(langTable[name] or rp_languages.en[name], ...) or nil
 end
 
+function DarkRP.getPhraseLocalized(name, ply, ...)
+    local lang = ply and ply:IsValid() and ply:GetInfo("gmod_language") or selectedLanguage
```

Comment:
It is possible that there is a local array of players, in which some player may be not valid (just nil value). And these are extra errors in the server console

Reply:
In this case, it's always down to your addon to have a list of valid players and not to gamemode to check that. In most cases, when an addon wants to call this function through all players, it will call `player.GetAll` and always return a list of valid players.

Reply:
I think it would be fine to just have `IsValid(ply)` to catch both `nil` and `NULL`, and not worry too much about the three extra instructions that it costs.